### PR TITLE
Add support for windows 64

### PR DESCRIPTION
**Pull request by melund**
The openjdk conda package has support for win64 so there is no reason this shouldn't work for win64 as well.

### CODE REVIEW

File: recipe/meta.yaml
Comment (carlodri) [inline recipe/meta.yaml]:
```suggestion

```

Comment (carlodri) [reply recipe/meta.yaml]: you can completely remove this line since only win64 is supported

File: recipe/meta.yaml
Comment (carlodri) [inline recipe/meta.yaml]:
```suggestion
  number: 1
```

increase the build number

**Review by carlodri — CHANGES_REQUESTED** [review]
thanks for this PR!

you also need to rerender the recipe so that the windows builds get triggered

**Comment by carlodri** [comment]
@JohnGreeley if you wish you can add me as a maintainer

**Comment by melund** [comment]
@conda-forge-admin please rerender

**Review by carlodri — APPROVED** [review]

**Comment by carlodri** [comment]
thanks @melund! We need to wait for @JohnGreeley...

**Comment by carlodri** [comment]
@conda-forge/core sorry for pinging, but I need this package in my workflow: can someone please merge? I can also be added as maintainer if desired.

**Comment by isuruf** [comment]
Let's give @JohnGreeley a few more days to review.

**Comment by scopatz** [comment]
This was opened back in August.  I'd be happy to merge now if you wanted to add either @carlodri or @melund to the maintainers list

**Comment by carlodri** [comment]
@scopatz that was why I opted to ping core, and also because @JohnGreeley seemed to be fairly inactive here on GH in the last years, at least from what you can tell from his profile. As I said, I'm happy to help in the maintenance if you want to add me, it's not a big chore and it would revive a very useful feedstock.

**Comment by scopatz** [comment]
It would be great if you could add yourself 😉

**Comment by carlodri** [comment]
@scopatz sure, will make a PR in a moment, sorry (I thought you intended to merge this PR after adding my name in it...)!

**Comment by scopatz** [comment]
You should be a maintainer now / soon @carlodri :) once you are added, feel free to merge this!

**Comment by carlodri** [comment]
Thanks!

**Comment by carlodri** [comment]
@conda-forge-admin, please rerender

**Pull request merged**
